app/utils/SessionKeys.scala

```diff
@@ -24,6 +24,7 @@ object SessionKeys {
   val dateCommunicationSent: String = "dateCommunicationSent"
   val penaltyId: String = "penaltyId"
   val reasonableExcuse: String = "reasonableExcuse"
+  val dateOfBereavement: String = "dateOfBereavement"
```

Comment:
There's an existing val whenDidThePersonDie to capture the date of bereavement, do we still need the val dateOfBereavement.